x/evm/keeper/utils_test.go

```diff
@@ -439,7 +440,7 @@ func (suite *KeeperTestSuite) TestVerifyFeeAndDeductTxCostsFromUserBalance() {
 			expectPassDeduct: true,
 			from:             suite.address.String(),
 			malleate: func() {
-				suite.ctx = suite.ctx.WithIsCheckTx(true)
+				suite.ctx = suite.ctx.WithIsCheckTx(true).WithConsensusParams(*app.DefaultConsensusParams)
```

Comment:
It seems unnecessary.

Reply:
Same 
- https://github.com/xpladev/ethermint/pull/14/files#diff-86a2788c6f900e3e8d070486fa1b357f309c85c581bb1b7b64597b7d7445cc8dR126
- https://github.com/xpladev/ethermint/pull/14/files#diff-86a2788c6f900e3e8d070486fa1b357f309c85c581bb1b7b64597b7d7445cc8dR149

Reply:
Same
- https://github.com/xpladev/ethermint/pull/14/files#diff-63855f036ffb07403867a5ce483596b316680c10e6034fe60304cc9a787be380R912
- https://github.com/xpladev/ethermint/pull/14/files#diff-63855f036ffb07403867a5ce483596b316680c10e6034fe60304cc9a787be380R1097
- https://github.com/xpladev/ethermint/pull/14/files#diff-63855f036ffb07403867a5ce483596b316680c10e6034fe60304cc9a787be380R1205

Reply:
https://github.com/xpladev/ethermint/pull/14/commits/0dd3e83b3d83abc20727e63bc0dcb5f5c3703dde